client/src/app/pages/profile/profile.component.ts

```diff
@@ -10,18 +11,24 @@ export class ProfileComponent {
 
   publications: Post[] = [];
 
-  constructor(private readonly profileService: ProfileService,) {
-    this.publications = this.profileService.getPublications();
-
+  constructor(private readonly postService: PostsService) {
+    this.postService.getAllPublicationsPerson(9).subscribe({
```

Comment:
Add a TODO comment here, so as to not forget to add the actual user ID from user service, instead of the hardcoded "9" ID.